protocol/query/method/message/data/dataElectionResult.json

```diff
@@ -19,10 +19,10 @@
 						"$comment": "ID of the question"
 					},
 					"result": {
-						"description": "[Array[String]] results",
+						"description": "[Array[object]] results",
 						"type": "array",
 						"items": {
-							"type": "string",
+							"type": "object",
```

Comment:
you might want to clarify the format of that object (i.e. which properties, etc.)

Reply:
Done